src/components/Claim/Generic/AlreadyClaimed.view.tsx

```
@@ -11,28 +11,23 @@ export const AlreadyClaimedLinkView = ({ claimLinkData }: { claimLinkData: inter
-            <label className="text-h8 font-bold ">
-                This link previously contained {claimLinkData?.tokenSymbol} on{' '}
-                {consts.supportedPeanutChains &&
-                    consts.supportedPeanutChains.find((chain) => chain.chainId == claimLinkData?.chainId)?.name}
-            </label>
-            <label className="text-h9 font-normal">
+            <div className="space-y-2">
+                <label className="text-h2">Sorry, this link has been claimed already.</label>
+                <label className="">
+                    This link previously contained {claimLinkData?.tokenSymbol} on{' '}
+                    {consts.supportedPeanutChains &&
+                        consts.supportedPeanutChains.find((chain) => chain.chainId == claimLinkData?.chainId)?.name}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)